observability/power_monitoring/power-monitoring-kepler-power-attribution-guide.adoc

```diff
@@ -0,0 +1,14 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="power-monitoring-kepler-power-attribution-guide_{context}"]
+= Power monitoring Kepler power attribution guide
+include::_attributes/common-attributes.adoc[]
+:context: power-monitoring-kepler-power-attribution-guide
+
+toc::[]
+
+:FeatureName: Power monitoring
+include::snippets/technology-preview.adoc[leveloffset=+2]
+
+Kepler's power attribution system provides practical, proportional distribution of hardware energy consumption to individual workloads. While CPU-time-based attribution has inherent limitations due to modern CPU complexity, it offers a good balance between accuracy, simplicity, and performance overhead for most monitoring and optimization use cases.
+
+For more information about power attribution, see link:https://github.com/sustainable-computing-io/kepler/blob/main/docs/developer/power-attribution-guide.md[Kepler Power Attribution Guide].
```

Comment:
Could we replace this with http://sustainable-computing.io/usage/power-attribution ?